test/TestUtilities/Test.Utility/Signing/CertificateUtilities.cs

```diff
@@ -27,7 +27,9 @@ internal static AsymmetricCipherKeyPair CreateKeyPair(int strength = 2048)
 
         internal static string GenerateFingerprint(X509Certificate certificate)
         {
+#pragma warning disable CS0618 // Type or member is obsolete
             using (var hashAlgorithm = CryptoHashUtility.GetSha1HashProvider())
+#pragma warning restore CS0618 // Type or member is obsolete
```

Comment:
The obsolete message says to use `GetSha1HashProvider`. Perhaps this test code should use that instead.